src/main/scala/com/typesafe/sbt/packager/archetypes/scripts/AshScriptPlugin.scala

```diff
@@ -69,9 +69,11 @@ import sbt._
   *  enablePlugins(AshScriptPlugin)
   * }}}
   */
+@deprecated("Use AshStartScriptPlugin", "1.4.1")
```

Comment:
I wonder why we need to do this. As mentioned in https://github.com/sbt/sbt-native-packager/issues/1264#issuecomment-548263952 I'm currently not seeing the benefit as we copy all settings, which are identical, but add the complexity of having to deal with versions of a script where actually only one should exist.

Reply:
I just wanted to rename so it is ashSTARTSCRIPTplugin not just ashSCRIPTplugin. I agree we can reuse keys, will change that.

Reply:
That makes sense :+1: